Index.html

```diff
@@ -31,7 +31,170 @@ <h3 class="connect">LET’S CONNECT</h3>
+          </nav>
+          <p class="proj-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="softwares">
+            <li class="software">html</li>
+            <li class="software">css</li>
+            <li class="software">javascript</li>
+          </ul>
+          <button class="btn">see project</button>
+        </div>
+
+      </div>
+
+      <div class="proj4">
+        <div class="proj-content-space">
+          <h1 class="proj-title">Multi-Post Stories</h1>
+          <nav class="proj-img2">
+            <h4 class="cpny">Canopy</h4>
+            <img src="./images/Counter.png" alt="counter" />
+            <span class="project-lang">Back End Dev</span>
+            <img src="./images/Counter.png" alt="counter" />
+            <span>2015</span>
+          </nav>
+          <p class="proj-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="softwares">
+            <li class="software">html</li>
+            <li class="software">css</li>
+            <li class="software">javascript</li>
+          </ul>
+          <button class="btn">see project</button>
+        </div>
+
+      </div>
+    </div>
+
+   </section> 
+   
+   <section>
+    <div class="myself">
+      <h2 class="name">About<br/>Myself</h2>
+      <p class="text">
+        Hello I’m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience! If you
+        like what you see and have a project you need coded, don’t hestiate to
+        contact me.
+      </p>
+      <h3 class="connect text">LET’S CONNECT</h3>
+        <ul class="socialmedia">
+          <li><a href="#"><img class="twitter" src="images\twitter.png" alt="twitter"></a></li>
+          <li><a href="#"><img class="i" src="images\linkedin.png" alt="i"></a></li>
+          <li><a href="#"><img class="m" src="images\M.png" alt="m"></a></li>
+          <li><a href="#"><img class="g" src="images\g.png" alt="g"></a></li>
+          <li><a href="#"><img class="v" src="images\v.png" alt="v"></a></li>
+        </ul>
+        <button class="resume">Get my resume</button>
+    </div>
+    <div>
+      <ul class="pro-languages">
+        <li>
+          <h4 class="tab-title displaytype">Languages</h4>
+          <a href="#">
+            <img src="./images/rightarrow.png" alt="right" class="right-arrow" />
+          </a>
+        </li>
+      </ul>
+      <ul class="frame-list">
+        <li class="frame">
+          <img
+            src="./images/js.png"
+            alt="js"
+            class="ellipse"
+          />
+          <h4 class="frame-name">Javascript</h4>
+        </li>
+        <li class="frame">
+          <img src="./images/html.png" alt="html" class="ellipse" />
+          <h4 class="frame-name">HTML</h4>
+        </li>
```

Comment:
Kindly also update the styling of your languages  section to better match the figma design🙏.
Your design:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/37116322/219809580-d985586f-11f7-4606-bb5a-1ecc9622a442.png">

Figma design:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/37116322/219809809-538619ce-e978-49ed-9b29-59c28eeab244.png">

A few things to note:
- The background color of this section should be white(#fff)
- The arrows on the right of each technology should be aligned
- The technologies(HTML, CSS, JS) should fit into the languages section and have a different background color

**Please try your best to make sure that your design matches the Figma design as much as possible😎**